web/geosearch/datapunt_geosearch/datasource.py

```diff
@@ -50,6 +50,12 @@ def __init__(self, dsn=None, connection=None):
 
         self.meta = self.metadata.copy()
 
+    @classmethod
+    def check_scopes(cls, scopes=None):
```

Comment:
```suggestion
    def check_scopes(cls, scopes=None):
        """Check who may access this datasource. When no scopes are given,
        access is allowed when there are no restrictions to the data.
        """
```

Also, perhaps rename `check_scopes()` into something like `is_authorized()`?